cmd/build/helmify/main.go

```diff
@@ -75,7 +75,7 @@ func (ks *kindSet) Write() error {
 			subPath = "crds"
 			parentDir := path.Join(*outputDir, subPath)
 			fmt.Printf("Making %s\n", parentDir)
-			if err := os.Mkdir(parentDir, 0750); err != nil {
+			if err := os.Mkdir(parentDir, 0o750); err != nil {
```

Comment:
I haven't seen this before, what's the `o` here?

Reply:
It's a new default for `gofumt`. The `o` explicitly denotes that the number is in octal format, similar to the `x` in `0x1234` for hex. We can disable this specific rule if we want.

I like the `0o` prefix since it means it's easier to spot mistakes like `os.Mkdir(parentDir, 750)`, but I don't feel strongly about it.

Reply:
Looks good, I was just curious about it 🙂